README.md

```diff
@@ -69,14 +69,14 @@ Clip is very straightforward to compile and run if you're already an iOS develop
 
 1. Clone the repository 
 	``` 
-	https://github.com/rileytestut/Clip.git
+	git clone https://github.com/rileytestut/Clip.git
 	```
 2. Update submodules: 
 	```
 	cd Clip 
 	git submodule update --init --recursive
 	```
-3. Open `Clip.xcworkspace` and select the Clip project in the project navigator. On the `Signing & Capabilities` tab, change the team from `Yvette Testut` to your own account.
+3. Open `Clip.xcodeproj` and select the Clip project in the project navigator. On the `Signing & Capabilities` tab, change the team from `Yvette Testut` to your own account.
 4. Build + run app! 🎉
```

Comment:
At least if you have a developer account, you need to change the bundle IDs for Clip, ClipBoard, and ClipboardReader, remove the `group.com.rileytestut.Clip` App Group, and add back your own. I also changed the group names in `ALTAppGroups` in the `Info.plist`s but that may not be necessary.

Reply:
For any future readers, I found changing the group names in the `ALTAppGroups` necessary – when not doing so, you're going to get group-related issues when the Database is initialized on application startup, akin to:
```
DatabaseManager.swift:42: Fatal error: Unexpectedly found nil while unwrapping an Optional value
```

Reply:
Even after updating the ALTAppGroups, I am still getting the same error:

<img width="780" alt="image" src="https://github.com/rileytestut/Clip/assets/34964694/93559515-b303-4902-9cd6-57cdaf486641">
<img width="747" alt="image" src="https://github.com/rileytestut/Clip/assets/34964694/06695842-dd4e-42cb-b95e-ddf6c019cb8c">
<img width="528" alt="image" src="https://github.com/rileytestut/Clip/assets/34964694/6be082ac-9b6b-43d1-a8b4-a38790d0e961">
<img width="527" alt="image" src="https://github.com/rileytestut/Clip/assets/34964694/bb575d06-a585-4b84-9678-95ec8588a2a0">